src/conversions.jl

```diff
@@ -648,7 +649,8 @@ end
 #                               Helper Functions                               #
 ################################################################################
 
-to_linspace(interval, N) = range(minimum(interval), stop = maximum(interval), length = N)
+to_linspace(interval::Interval, N) = range(leftendpoint(interval), stop = rightendpoint(interval), length = N)
+to_linspace(x, N) = range(first(x), stop = last(x), length = N)
```

Comment:
@SimonDanisch @jkrumbiegel Is it fine to switch away from `minimum` and `maximum` to enable reverse image/heatmap axes? Should we consider this breaking?

(This is fixing the points I made in the last comment)

Reply:
If this change is fine we can merge this

Reply:
Is that in the code path, that already warns for vectors?

Reply:
Yeah I guess it is... So we don't allow vectors for images anymore anyways 🤷 
It's not yet erroring, but should already warn to not use it.
So shouldn't be breaking

Reply:
We have two functions with that change here:
- `to_interval` affecting Image, Volume (errors in data_limits with reversed axis though)
- `to_linspace` affecting PointBased w/ (Interval, Interval, Matrix), heatmap

What changed as far as I can tell:
- heatmap no longer errors for reverse intervals but otherwise works the same (is the RangeLike convert not getting hit?)
- image no longer errors on reverse intervals and allows for reversing, which it previously didn't. This is visually different
- PointBased no longer errors on for that specific input, but still with something like (Interval, Vector)
- volume errors in data_limits instead of convert_arguments with reversed limits